src/redis-cli.c

```diff
@@ -3023,6 +3038,7 @@ static void usage(int err) {
 "  --askpass          Force user to input password with mask from STDIN.\n"
 "                     If this argument is used, '-a' and " REDIS_CLI_AUTH_ENV "\n"
 "                     environment variable will be ignored.\n"
+"  --readpass <filename> Read password from file.\n"
```

Comment:
Maybe put `--readpass` after `--pass`, before `--askpass`?

At `--user` above, change "Needs -a" to "Needs -a, --pass, --readpass or --askpass" or simply "Needs password".

Reply:
it's a good idea.  thank you for you advices